x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -166,7 +167,40 @@ export async function getKibanaAssets(
   return result;
 }
 
-async function installKibanaSavedObjects({
+const isImportConflictError = (e: SavedObjectsImportFailure) => e?.error?.type === 'conflict';
+/**
+ * retry saved object import if only conflict errors are encountered
+ */
+async function retryImportOnConflictError(
+  importCall: () => ReturnType<SavedObjectsImporterContract['import']>,
+  {
+    logger,
+    maxAttempts = 5,
+    _attempt = 0,
+  }: { logger?: Logger; _attempt?: number; maxAttempts?: number } = {}
+): ReturnType<SavedObjectsImporterContract['import']> {
+  const result = await importCall();
+  const errors = result.errors ?? [];
+
+  if (_attempt < maxAttempts && errors.length && errors.every(isImportConflictError)) {
+    const retryCount = _attempt + 1;
+    const retryDelaySec = Math.min(Math.pow(2, retryCount), 64); // @dover's patented retry, 2s, 4s, 8s, 16s, 32s, 64s, 64s, 64s ...
```

Comment:
I'm not sure that an _exponential_ delay here really helps us much since these are likely to all be running on the same node and the issue isn't that Elasticsearch is under high load. Exponential backoffs are usually useful when there's a load issue and we want to avoid creating a cascading failure.

I also think that one thing that might (?) help more would be to add some random "jitter" to the delay so that if 100 packages hit a conflict at once, they don't all retry at the same time and are more likely to have more succeed.

What do you think about using a 1s delay + random jitter of 0-3000ms?